src/matrix.js

```diff
@@ -1119,6 +1119,18 @@ export class AbstractMatrix {
     return result;
   }
 
+  kroneckerSum(other) {
+    other = Matrix.checkMatrix(other);
+    if (!this.isSquare() || !this.isSquare()) {
+      throw new Error('Kronecker Sum needs two Square Matrices');
+    }
+    let m = this.rows;
+    let n = other.rows;
+    let AxI = this.kroneckerProduct(Matrix.eye(n, n));
+    let IxB = Matrix.eye(m, m).kroneckerProduct(other);
+    return Matrix.add(AxI, IxB);
```

Comment:
```suggestion
    return AxI.add(IxB);
```

I think this should still work, will avoiding to create a third matrix.

Reply:
Thank you for your review!